app/components/fandom-app-smart-banner.js

```diff
@@ -70,7 +71,7 @@ export default Component.extend({
 	 */
 	checkForHiding() {
 		if (!standalone && $.cookie('fandom-sb-closed') !== '1') {
-			this.sendAction('toggleVisibility', true);
+			this.get('smartBanner').setBannerVisibility(true);
```

Comment:
setVisibility would be enough since this service name is smartBanner :) (not a blocker)